versioned_docs/version-v2.1.4/token/token-wrapping.md

```
@@ -0,0 +1,73 @@
+
+   ![Staking hub wrapper](/img/token/wrapper-3.jpg)
+
+## 5. Confirm the transaction
+
+   After entering the desired amount, click "**Swap**" to initiate the wrapping process. Confirm the transaction in your wallet when prompted. Once the transaction is confirmed, you can import the wrapped HOPR token (**wxHOPR**) into your wallet by using the correct smart contract address, which can be found [here](./acquiring-hopr-tokens.md#hopr-tokens-on-different-chains).
+
+   ![Staking hub wrapper](/img/token/wrapper-4.jpg)
+</TabItem>
+<TabItem value="unwrapp" label="Unwrap wxHOPR to xHOPR">
+
+## 1. Navigate to the Staking Hub
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)